packages/twenty-front/src/modules/object-record/record-table/record-table-row/components/RecordTableTr.tsx

```diff
@@ -56,6 +139,13 @@ export const RecordTableTr = forwardRef<
         data-virtualized-id={recordId}
         isDragging={isDragging}
         ref={ref}
+        data-active={isActive ? 'true' : 'false'}
+        data-focused={
+          isRowFocusActive && isFocused && !isActive ? 'true' : 'false'
+        }
+        data-next-row-active-or-focused={
+          isNextRowActiveOrFocused ? 'true' : 'false'
+        }
```

Comment:
Great to see HTML data attributes! You can directly use booleans as values for HTML attributes. React will coerce them to strings.

```suggestion
        data-active={isActive}
        data-focused={isRowFocusActive && isFocused && !isActive}
        data-next-row-active-or-focused={isNextRowActiveOrFocused
```

Reply:
I tried and it didn't work, apparently a false boolean will result in the attribute not being set.